doc/Administration-guide/System-installation/Cluster-setup/index.md

```diff
@@ -5,62 +5,115 @@ permalink: /:path/
 sort_idx: 30
 ---
 
-* [Introduction](#introduction)
-* [Cluster configuration](#cluster-configuration)
-  * [List of properties](#list-of-properties)
-  * [Kaa-node configuration](#kaa-node-configuration)
-  * [Zookeeper configuration](#zookeeper-configuration)
-  * [SQL database configuration](#sql-database-configuration)
-  * [NoSQL database configuration](#nosql-database-configuration)
-  * [Firewall rules configuration](#firewall-rules-configuration)
+{% include variables.md %}
 
-## Introduction
+* TOC
+{:toc}
+
+This guide describes configuration of Kaa cluster on Linux nodes.
+
+The guide contains instructions on how to configure kaa-node service and required third party components like Apache Zookeeper service, SQL and NoSQL databases.
+
+## Requirements
+
+In order to set up Kaa cluster you need to have at least 3 Linux nodes with ```kaa-node``` service installed on each of them to create a reliable cluster, refer to [Single node installation]({{root_url}}Administration-guide/System-installation/Single-node-installation/) guide for installation details. 
 
-This page describes how to setup and configure kaa-node cluster.
+Also ```kaa-node``` service require some third party dependencies like SQL and NoSQL databases and Apache Zookeeper service, more details you can find in [Architecture overview]({{root_url}}Architecture-overview/). 
+You can find detailed instructions on how to install and configure Zookeeper service and one of supported SQL and NoSQL databases in [Single node installation]({{root_url}}Administration-guide/System-installation/Single-node-installation/#installation-steps) guide. 
+A set of databases (for example MongoDB + PostgreSQL) depends on your particular use case and some useful recommendations you can find in [Planning your deployment]({{root_url}}Administration-guide/System-installation/Planning-your-deployment/) guide.
 
-In general cluster setup is similar to [single node setup](../Single-node-installation), except few details. We need at least 3 nodes to create a reliable cluster. For simplicity in this tutorial we will be using single instance of SQL and NoSQL databases, so this tutorial doesn't cover such themes like setting up Cassandra, MongoDB or PostgreSQL cluster setup. Let us assume that we have 3 nodes with Ubuntu 14.04 installed on each of them.
+In this guide we assume that you had already set up your SQL and NoSQL database clusters, so this tutorial doesn't cover such themes like setting up ones for Cassandra, MongoDB or PostgreSQL. 
+Refer to official [Cassandra](http://docs.datastax.com/en/landing_page/doc/landing_page/current.html), [MongoDB](https://docs.mongodb.com/manual/) and [PostgreSQL](https://www.postgresql.org/docs/) documentation in order to setup corresponding database cluster.
+
+MariaDB come with out of the box master-master replication support thus we recommend to use this database in your Kaa cluster. 
```

Comment:
"with out of the box" need to be replaces with "with out-of-the-box" to avoid confusion during reading.